.changeset/good-rivers-add.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/cloudflare': minor
+---
+
+Improves default config to support `cloudflare:sockets` imports
```

Comment:
nit: since it's a `minor`, maybe the changeset should contain more information e.g. what it means for the user and they can do now

Reply:
Is it even a breaking change? I don't see how this would break any case that was working. It might change the error in some scenarios, but not add new ones.